samples/exposed-migration/src/main/kotlin/Application.kt

```diff
@@ -0,0 +1,60 @@
+import org.flywaydb.core.Flyway
+import org.jetbrains.exposed.sql.Database
+import org.jetbrains.exposed.sql.ExperimentalDatabaseMigrationApi
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.insert
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.jetbrains.exposed.sql.vendors.currentDialect
+import java.util.*
+
+const val URL = "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1"
+const val USER = "root"
+const val PASSWORD = ""
+
+@OptIn(ExperimentalDatabaseMigrationApi::class)
+fun main() {
```

Comment:
What is the purpose of this main function? Who will run it? Would it better to create a test and check that a correct migration is created after running such a script?

Reply:
The purpose of the `main` function is to make the project runnable. A user who wants to play around with the sample would run it.
My understanding is that a sample project simply demonstrates usage of a particular feature or set of features, whereas the tests for the features would be in the `exposed-tests` module. Is that correct? If not, I'm happy to modify it.

Reply:
Should we make a separate task (like `generateMigrations`) and function separate from the main to do that? It would be really nice. So we can have an app in `main` and some other function (and task) for migrations.

Feel free to make it as a separate PR